mobile/lib/src/domain/sleep_sequence/i_sleep_sequence_repository.dart

```diff
@@ -3,5 +3,7 @@ import 'package:polydodo/src/domain/sleep_sequence/sleep_sequence_stats.dart';
 abstract class ISleepSequenceRepository {
   List<SleepSequenceStats> getSleepSequences();
 
+  void addSleepSequence(SleepSequenceStats sequence);
```

Comment:
On voit déjà que l'argument est une sleepSequence.

```suggestion
  void store(SleepSequenceStats sequence);
```